config.example.toml

```diff
@@ -1,50 +1,43 @@
 # Controls deployment options
 [deployment]
-    host = "127.0.0.1"
+    host = "0.0.0.0" # Set to 127.0.0.1 if you use a reverse proxy, or access it locally
     port = 8000
 
     # Amount of worker Priviblur instances to spawn. Increases speed significantly.
     # workers = 1
 
-    # # If you're running Priviblur behind a remote proxy, one or more of the following must be set
-    # # can also be set via env variables by captialzing and prefixing with PRIVIBLUR_
-    # #
-    # # For more information see
-    # # https://sanic.dev/en/guide/advanced/proxy-headers.html
-    # #
-    # # Default: None
-    # #
-    # forwarded_secret =
-    # real_ip_header =
-    # proxies_count =
-
-# Controls redis cache options
-# Ignore to disable the cache
-#
+
+# # Controls redis cache options
+# # Ignore to disable the cache
+# # 
+# # To enable caching, Redis is required, you then have to uncomment "url" and set it accordingly. Every other options in this section will use their default value (indicated next to them) if they are kept commented out.
 # [cache]
+
     # url =
+    # # For docker use the following:
```

Comment:
Shouldn't the url for docker be double-commented here?

Reply:
it is?

Reply:
Oops, I added the review to the wrong line. I meant this

https://github.com/syeopite/priviblur/blob/f032853e55848673f570abbec8b9b149925fb7d4/config.example.toml#L18

Reply:
Well, as I said:

>I just made it so users just have to uncomment it instead of the first one